pkg/token/azurecli.go

```diff
@@ -0,0 +1,72 @@
+package token
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/go-autorest/autorest/adal"
+)
+
+type azureCLIToken struct {
+	clientID    string
```

Comment:
clientID is not used. please remove it

Reply:
what happens is, it's using the azure-cli's client instead.

Reply:
since clientID is not used, can you also remove it in `convert.go`?

Reply:
Actually, tenant ID wasn't used either, because you can only be signed in using one tenant. Removed that too, all the way through.